### PR TITLE
Fix Example and Add Support for Connection String

### PR DESCRIPTION
**Pull request by lsabi**
- Add support for Connection String  #[https://github.com/iggy-rs/iggy-python-client/issues/22](https://github.com/iggy-rs/iggy-python-client/issues/22)
- Simplify client to support `stream` and `topic` names instead of ids
- Fix examples according to changes

### CODE REVIEW

**Comment by spetz** [comment]
Thank you for the contribution, LGTM :)

**Review by spetz — DISMISSED** [review]

**Comment by lsabi** [comment]
@spetz You're welcome. Two notes though:

- I thought that it's more pythonic to have functions like `create_stream("sample-stream")` `create_topic("stream-name", "topic-name")`. Maybe I could add a more "low level" layer that uses also `IDs`
- Do you mind testing it? I've tested on my local, but then copied and pasted the edits on the online editor. Hope I didn't mess it up...

**Comment by spetz** [comment]
@lsabi would you mind joining our [Discord](https://iggy.rs/discord)? I'm not a Python dev myself, so hopefully the other members could be more helpful. On my side, I'm open to any suggestions that you have :)

File: src/client.rs [hunk 1]
```diff
@@ -50,15 +51,21 @@ impl IggyClient {
...
         let runtime = Builder::new_multi_thread()
             .worker_threads(4) // number of worker threads
             .enable_all() // enables all available Tokio features
             .build()
             .unwrap();
+    let client = IggyClientBuilder::new()
+        .with_tcp()
+        .with_server_address(conn.unwrap_or("127.0.0.1:8090".to_string()))
+        .build()
+		.unwrap();
```
Comment (FireMasterK) [inline src/client.rs:66]: I would recommend using this:
```
IggyClient::from_connection_string(&iggy_address)
```

Iggy has its own connection string like `iggy://iggy:iggy@127.0.0.1:8090`.

Comment (spetz) [reply src/client.rs:66]: The example using TCP is expecting the raw hostname + port, while the `IggyClient` (which is an additional wrapper on top of the selected transport layer) supports the custom connection string.

**Comment by mmodzelewski** [comment]
> * Do you mind testing it? I've tested on my local, but then copied and pasted the edits on the online editor. Hope I didn't mess it up...

@lsabi Please let me know once the PR is ready, I'll be able to test it.

**Review by mmodzelewski — APPROVED** [review]

**Comment by lsabi** [comment]
@mmodzelewski does it work on your computer?

I was still testing this code, since I found a problem with the `Identifier` parameter of `create_topic`, `send_messages` and `poll_messages`...it seems that it changes the ID on every new connection. Or it doesn't associate the name with the id, resulting in topics with the same name, but different ID.

**Comment by mmodzelewski** [comment]
@lsabi yes, it worked fine, so I merged it. If I try to create a stream or topic with the same name, I get an error, so that looks good. I'll do some more testing later today, but if you have any specific use case that you would like me to try out, let me know.

**Comment by lsabi** [comment]
@mmodzelewski now it seems to be working on my computer as well.

Well enough, it could be deployed to the public package registry....am I supposed to change the version from `0.2.4` to `0.2.5`? If so, where exactly? I've never worked with `PyO3`

**Comment by mmodzelewski** [comment]
@lsabi great to hear that it works for you as well :)
I'll add another small change and will push a new version, probably tomorrow.

**Pull request merged**